gemini_report_writer/agents/retriever.py

```
@@ -10,58 +10,140 @@ def __init__(self, redis_host='localhost', redis_port=6379, redis_db=0):
+                    authors = [author.get('display_name') for author in work.get('authorships', []) if author.get('display_name')]
+                    year = work.get('publication_year')
+                    citations = work.get('cited_by_count', 0)
+
+                    primary_location = work.get('primary_location') or {}
+                    source_info = primary_location.get('source') or {}
+                    journal_name = source_info.get('display_name')
+
+                    biblio = work.get('biblio') or {}
+                    first_page = biblio.get('first_page')
+                    last_page = biblio.get('last_page')
+                    pages = f"{first_page}-{last_page}" if first_page and last_page else None
```

Ruff (0.11.9)

Local variable `pages` is assigned to but never used

Remove assignment to unused variable `pages`

(F841)